Jenkinsfile

```diff
@@ -68,6 +68,13 @@ void buildManifest(String variant_name, String bitbake_image) {
             }
         }
 
+        if ("smoke-test" in options || "export-oe-tests" in options) {
```

Comment:
In another project we use a Set called "options" which is passed as a parameter to buildManifest. That Set it not available here and that's what the build is failing on.

Reply:
Thanks! I realized that this is why is fails, but was trying to find "options". Do you have some time for help me, @erikboto? @rpannek started to help me but is not in the office now.